lte/gateway/c/core/oai/tasks/nas/emm/EmmCommon.c

```diff
@@ -500,3 +501,26 @@ void create_new_attach_info(
   emm_context_p->new_attach_info->is_mm_ctx_new  = is_mm_ctx_new;
   OAILOG_FUNC_OUT(LOG_NAS_EMM);
 }
+
+partial_list_t* emm_verify_orig_tai(const tai_t orig_tai) {
+  partial_list_t* par_list = NULL;
+
+  for (uint8_t list_i = 0; list_i < mme_config.num_par_lists; list_i++) {
+    if (!mme_config.partial_list) {
+      OAILOG_ERROR(LOG_NAS_EMM, "partial_list in mme_config is NULL\n");
+      OAILOG_FUNC_RETURN(LOG_NAS_EMM, par_list);
+    }
```

Comment:
should not this check be done before the for loop instead of checking it every time inside the loop?